README.md

```diff
@@ -1,2 +1,70 @@
-# react-todo-app
-A simple To-Do app built with React (create app). This app allows users to add and delete tasks to organize their days.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
```

Comment:
[Required] :recycle:      Please ensure that you make this Readme descriptive. This is the first landing of users as well as other people who may want to see your project. Also give acknowledgement if anyone's code was used. This ensures that you avoid potential copyright problems.

Reply:
@Nyame-Wolf Thanks for pointing this out :+1: